plat/rpi3/include/platform_def.h

```diff
@@ -181,6 +181,14 @@
 #define BL32_DRAM_BASE			SEC_DRAM0_BASE
 #define BL32_DRAM_LIMIT			(SEC_DRAM0_BASE + SEC_DRAM0_SIZE)
 
+#if LOAD_IMAGE_V2
```

Comment:
You don't need the ``#if LOAD_IMAGE_V2``, this port doesn't support LOAD_IMAGE_V1 (it is deprecated). Check ``plat/rpi3/platform.mk`` lines 82 and 112.

Reply:
You haven't removed this one.